astroid/node_classes.py

```diff
@@ -1053,15 +1053,17 @@ def postinit(self, targets=None, value=None):
 
 class AnnAssign(mixins.AssignTypeMixin, Statement):
     """class representing an AnnAssign node"""
-    _astroid_fields = ('target', 'value', 'annotation',)
+    _astroid_fields = ('target', 'annotation', 'value', 'simple')
     target = None
-    value = None
     annotation = None
+    value = None
+    simple = None
 
-    def postinit(self, target=None, value=None, annotation=None):
+    def postinit(self, target=None, annotation=None, value=None, simple=None):
```

Comment:
Something I just noticed: which of these fields are optional in the standard-library ast nodes?  I'm assuming that the annotation, target, and value fields aren't, but I haven't looked at 3.6 myself.  Whatever fields aren't optional should have required arguments.  There's legacy code that requires that it be possible to build some AST nodes without having all the mandatory fields set, but this is bad practice, and we shouldn't be carrying it forward to new nodes.